src/apps/uniswap/positions.ts

```diff
@@ -53,6 +54,81 @@ const UNI_V3_ADDRESSES_BY_NETWORK_ID: {
   [NetworkId['celo-alfajores']]: undefined,
 }
 
+export async function getUniswapV3PositionDefinitions(
+  networkId: NetworkId,
+  address: Address,
+  userPositionsMulticall: Address,
+  nftPositions: Address,
+  factory: Address,
+): Promise<ContractPositionDefinition[]> {
+  const client = getClient(networkId)
+  const userPools = await client.readContract({
+    abi: userPositionsAbi,
+    address: userPositionsMulticall,
+    functionName: 'getPositions',
+    args: [nftPositions, factory, address as Address],
+  })
+
+  return userPools
+    .map((pool) => ({
+      ...pool,
+      token0: pool.token0.toLowerCase(),
+      token1: pool.token1.toLowerCase(),
+    }))
+    .filter((pool) => pool.liquidity > 0)
+    .map((pool) => {
+      return {
+        type: 'contract-position-definition',
+        networkId,
+        address: pool.poolAddress,
+        tokens: [
+          { address: pool.token0, networkId },
+          { address: pool.token1, networkId },
+        ],
+        displayProps: ({ resolvedTokensByTokenId }) => ({
+          title: `${
+            resolvedTokensByTokenId[
+              getTokenId({
+                address: pool.token0,
+                networkId,
+              })
+            ].symbol
+          } / ${
+            resolvedTokensByTokenId[
+              getTokenId({
+                address: pool.token1,
+                networkId,
+              })
+            ].symbol
+          }`,
+          description: 'Pool',
+          imageUrl:
+            'https://raw.githubusercontent.com/valora-inc/dapp-list/ab12ab234b4a6e01eff599c6bd0b7d5b44d6f39d/assets/uniswap.png',
+        }),
```

Comment:
Looks like we should make this a parameter, so the Ubeswap V3 pools still use the Ubeswap logo, right?

Reply:
Oops, my bad